packages/cpt-ui/src/pages/BasicDetailsSearchResultsPage.tsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable max-len */
+import React from "react"
+import {useNavigate} from "react-router-dom"
+import {
+  BackLink,
+  Table,
+  Container,
+  Row,
+  Col
+} from "nhsuk-react-components"
+import {SearchResultsPageStrings} from "@/constants/ui-strings/BasicDetailsSearchResultsPageStrings"
+import {FRONTEND_PATHS} from "@/constants/environment"
+
+// Mock patient data (from the prototype)
+const patients = [
+  {
+    nhsNumber: "9726919207",
+    given: "Issac",
+    family: "Wolderton-Rodriguez",
+    gender: "Male",
+    dateOfBirth: "6-May-2013",
+    address: "123 Brundel Close, Headingley, Leeds, West Yorkshire, LS6 1JL"
+  },
+  {
+    nhsNumber: "9725919207",
+    given: "Steve",
+    family: "Wolderton-Rodriguez",
+    gender: "Male",
+    dateOfBirth: "6-May-2013",
+    address: "123 Brundel Close, Headingley, Leeds, West Yorkshire, LS6 1JL"
+  }
+]
+
+export default function SearchResultsPage() {
+  const navigate = useNavigate()
+
+  // to sort by first name
+  const sortedPatients = patients
```

Comment:
We know what form the data will take from the backend, so rather than only having mock data here, we should have the logic here to make that call and only fall back on the mock data for specific cases, and even then only if the proper call fails. See [here](https://github.com/NHSDigital/eps-prescription-tracker-ui/blob/main/packages/cpt-ui/src/pages/PrescriptionDetailsPage.tsx#L57) for an example

Reply:
Also, the type that gets returned from the backend should be defined in the common-types package, if it isn't already.